### ISSUE TITLE
move-joints.py Hip yaw pitch

### ISSUE THREAD

**Issue by foxcub99** [UNSWComputing/PyBullet-Nao-Motion-Editor#4]
Currently only an option for LHipYawPitch and no option for RHipYawPitch meaning it cannot be changed. Needs to be changed to just HipYawPitch since they are linked and can only move together and should also move the right hipyawpitch at the same time.

**Comment by n33r4j** [comment]
> should also move the right hipyawpitch at the same time.

You mean they're not moving together when you move the slider? Seems to be working for me. Are you having any lag/performance issues?
![comp-HipYawPitch](https://github.com/UNSWComputing/PyBullet-Nao-Motion-Editor/assets/52103565/55bdff9b-3165-4bbb-870a-e387bae0a582)

> Needs to be changed to just HipYawPitch since they are linked and can only move together

Yep, you're right that they're linked but the name LHYP is maintained for convention, since that's how it is in the docs (both from [Aldebaran](http://doc.aldebaran.com/2-8/family/nao_technical/lola/actuator_sensor_names.html) and rUNSWift). HYP might be ambiguous, but I see where you're coming from. Also, I think it was the LHYP that was the dominant one (since it is powered while RHYP wasn't?).

**Comment by foxcub99** [comment]
Yeah getting quite a bit of lag, also not sure how to rotate the view

https://github.com/UNSWComputing/PyBullet-Nao-Motion-Editor/assets/9413043/3fe8a9c4-3b37-4f61-ad11-3006c5855c22

**Comment by pzrq** [comment]
@n33r4j  Could you please check your Aldebaran link? It doesn't seem to be working for me.

P.S. Absent other sources (perhaps it's hard-coded in C or otherwise at some low level?), I'd suggest going with [Softbank's v2.8 HipYawPitch](http://doc.aldebaran.com/2-8/family/nao_technical/motors_naov6.html). Older Nao versions, e.g. [for v2.0.6.8](http://doc.aldebaran.com/2-0/family/nao_dcm/actuator_sensor_names.html) surely aren't relevant to the Nao v6? If they are, it'd be good to know why, thanks.

**Comment by n33r4j** [comment]
> Could you please check your Aldebaran link? It doesn't seem to be working for me.

Just updated, sorry.(I left a leading exclamation mark)

I've linked to the same source, just a different page. To quote them,
> LHipYawPitch and RHipYawPitch are physically just one motor so they cannot be controlled independently. In case of conflicting orders, LHipYawPitch always takes the priority.
- [source](http://doc.aldebaran.com/2-8/family/nao_technical/joints_naov6.html)

It's pretty much LHYP everywhere else as well [rUNSWift](https://github.com/UNSWComputing/rUNSWift/blob/main/robot/utils/bodyV6.hpp), [qibullet](https://github.com/softbankrobotics-research/qibullet/blob/master/qibullet/robot_data/nao.urdf), etc.
